frontend/src/actions/TableActions.js

```diff
@@ -190,6 +184,68 @@ export function createTableData(rawData) {
 
 // THUNK ACTION CREATORS
 
+/*
+ * @method updateGridTable
+ * @summary Populate grid table with data and initial settings
+ *
+ * @param {string} tableId - table id
+ * @param {object} tableResponse - response data for the table
+ */
+export function fetchAttributes(tableId, tableResponse) {
+  return (dispatch, getState) => {
+    const state = getState();
+
+    // this check is only for unit tests purposes
+    if (state.tables) {
```

Comment:
Since there are tests that need to be adapted probably we can ditch this part and we can write the fe code without adding specific code to handle tests